src/ir/auth_logic/ast_construction_test.cc

```diff
@@ -40,9 +40,20 @@ struct QueryTestData {
   datalog::Predicate predicate;
 };
 
-class AstConstructionTest : public testing::TestWithParam<QueryTestData> {};
+struct SaysAssertionTestData {
+  std::string input_text;
+  std::string principal_name;
+  std::string fact_assertion_name;
+  datalog::Predicate cond_assertion_predicate;
+  datalog::Predicate rvalue;
+};
+
+class QueryAstConstructionTest : public testing::TestWithParam<QueryTestData> {
+};
+class SaysAssertionAstConstructionTest
+    : public testing::TestWithParam<SaysAssertionTestData> {};
```

Comment:
Move this to after `after the INSTANTIATE_TEST_SUITE_P(QueryAstConstructionTest, ...)`